src/entity/Invite.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Entity,
+  PrimaryColumn,
+  Column,
+  ManyToOne,
+  Index,
+  Generated,
+} from 'typeorm';
+import { User } from './User';
+import { FeatureType } from './Feature';
+
+export enum CampaignType {
+  Search = 'search',
+}
+
+@Entity()
+@Index('IDX_invite_userId_campaign', ['userId', 'campaign'], { unique: true })
+export class Invite {
+  @PrimaryColumn({ type: 'uuid' })
+  @Generated('uuid')
+  token: string;
+
+  @Column({ type: 'text' })
+  campaign: CampaignType | FeatureType;
+
+  @Column({ length: 36 })
```

Comment:
I think we are missing the `onDelete` property here. So when the user deletes their action we will set this value to null. The value should be along the lines of `onDelete: 'SET NULL'`. Kindly verify with the docs.